mlir/utils/performance/perfRunner.py

```diff
@@ -153,21 +197,46 @@ def getMetricArgsForRocprof(arch):
 # Bank conflict functions.The percentage of GPUTime LDS is stalled by bank
 # conflicts. Value range: 0% (optimal) to 100% (bad).
 def getBankConflict(fileName):
+    pass
+
+def getBankConflictV1(fileName):
     if not os.path.exists(fileName):
-        result = "NaN"
-        return result
+        return np.nan
     with open(fileName, 'r') as csv_file:
         reader = csv.DictReader(csv_file, delimiter = ',')
         header = reader.fieldnames
         if 'LDSBankConflict' not in header:
             return np.nan
-
-        result = []
+        sum = 0
```

Comment:
Counting the rows and accumulating the sum, vs accumulating a list and then calling sum and len on it.